p90.zone/dist/metadata.json

```diff
@@ -0,0 +1,22 @@
+{
+	"author": {
+		"name": "fror",
+		"id": "515668127829458945"
+	},
+	"service": "p90.zone",
+	"description": "p90.zone is a website that shows random WebMs (mostly memes) uploaded by their users",
```

Comment:
Doesnt this need to be "en:"

Reply:
en is only necessary when it has more than a translation